### ISSUE TITLE
Notification that an installation is waiting on another task to finish

### ISSUE THREAD

**Issue by stacycockrum** [freedombox/Plinth#377]
When installing applications and there other tasks running, such as a manual upgrade, it's difficult for the user to discern if the installation has stalled or if there is another task running.  It would be helpful if there was a message or indicator that provides a status.

First issue filed, please let me know if I should provide additional information.

**Comment by JoKeyser** [comment]
Hi @stacycockrum , could you comment to what extend #625 is a duplicate? Just to clarify and maybe close one of them.

**Comment by stacycockrum** [comment]
Hi @JoKeyser 
Thank you for following up on this.  However, I'm embarrassed to say that I can't remember exactly what was going on with this.  I'm sorry that I can't add anything else.  I'll close this one out.

**Comment by JoKeyser** [comment]
Hi, okay cool that it's a duplicate then. We'll follow it up in the other issue. Of course keep opening issues, no harm done by that, hopefully somebody will sort them :).
